akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -419,7 +419,7 @@ private[http] object HttpServerBluePrint {
           val isEarlyResponse = messageEndPending && openRequests.isEmpty
           if (isEarlyResponse && response.status.isSuccess)
             log.warning(
-              "Sending an 2xx 'early' response before end of request was received... " +
+              s"Sending an 2xx 'early' response before end of request for ${requestStart.uri.path} received... " +
```

Comment:
Maybe print the full URI? The host could be relevant if someone uses multi-homing.

```suggestion
              s"Sending an 2xx 'early' response before end of request for ${requestStart.uri} received... " +
```

Reply:
Good point